modules/apps/marketplace/marketplace-app-manager-web/src/main/java/com/liferay/marketplace/app/manager/web/internal/util/AppDisplayFactoryUtil.java

```diff
@@ -243,7 +243,13 @@ protected static void filterAppDisplays(
 		while (iterator.hasNext()) {
 			AppDisplay appDisplay = iterator.next();
 
-			if ((state > 0) && (appDisplay.getState() != state)) {
```

Comment:
Looks like you've removed the `state > 0` condition.  Was this intentional?  If not, why is this no longer needed?